backend/compilers/download.sh

```diff
@@ -40,6 +40,15 @@ if [[ "$uname" != "Darwin" ]]; then
     find . -name "CC1PSX.EXE" | xargs chmod +x
 fi
 
+# GCC mipsel
+if [[ "$uname" != "Darwin" ]]; then
```

Comment:
Would be nicer if the package had them already executable 🙂

Reply:
yeah that'd save every consumer of this the trouble